tdrs-backend/tdpservice/users/admin.py

```diff
@@ -20,7 +20,7 @@ class Meta:
 class UserAdmin(admin.ModelAdmin):
     """Customize the user admin functions."""
 
-    exclude = ['password', 'user_permissions']
+    exclude = ['password', 'user_permissions', 'is_active']
```

Comment:
Hides the `Active` form item for the User model in Django Admin, since we're using `deactivated` to handle this functionality.